tls/ttls/platform.c

```diff
@@ -37,7 +38,9 @@
 #if !defined(MBEDTLS_PLATFORM_STD_CALLOC)
 static void *platform_calloc_uninit( size_t n, size_t size )
 {
-    return vmalloc(n * size);
+    void *p = vmalloc(n * size);
+    memset(p, 0, n * size);
+    return p;
```

Comment:
There is ```vzalloc()``` call. Can we use ```kmalloc()``` instead? And probably we should use ATOMIC call. Anyway, verbose comment is required here. Probably we'll have many issues with the allocation when use it in Tempesta.